### PR TITLE
change priority list options

### PR DESCRIPTION
**Pull request by elena-vi**

### CODE REVIEW

**Comment by elena-vi** [comment]
[Rework - Emergency repair identification (priority list page + options) (MVP)](https://trello.com/c/ng7iuRnL/228-rework-emergency-repair-identification-priority-list-page-options-mvp)

File: compoments/radioFieldSet.js [hunk 1]
```diff
@@ -53,17 +54,36 @@ class RadioFieldSet extends Component {
...
-                  {o.title}
-                </label>
-              </div>
-            ))}
+            {this.options.map((o, i) => {
+              if (this.includeOrDivider && i == (this.options.length - 1)) {
+                return (
+                  <div>
+                    <div className="govuk-radios__divider">or</div>
+                    <div className="govuk-radios__item" key={i}>
```
Comment (Bhavesh-Patel) [inline compoments/radioFieldSet.js:62]: This block (lines 62-70) and the block in the else clause (lines 75-83) are essentially repeated.
Try to remove the duplication.

File: compoments/radioFieldSet.js [hunk 2]
```diff
@@ -53,17 +54,36 @@ class RadioFieldSet extends Component {
...
-                  id={`${this.name}-${i}`} name={this.name}
-                  type="radio" value={o.value} defaultChecked={o.checked}/>
-                <label className="govuk-label govuk-radios__label"
-                  htmlFor={`${this.name}-${i}`}>
-                  {o.title}
-                </label>
-              </div>
-            ))}
+            {this.options.map((o, i) => {
+              if (this.includeOrDivider && i == (this.options.length - 1)) {
```
Comment (Bhavesh-Patel) [inline compoments/radioFieldSet.js:58]: This check is being performed for every item, but it'll only be true for the last item - consider refactoring to improve flow of code.

File: compoments/radioFieldSet.js [hunk 3]
```diff
@@ -53,17 +54,36 @@ class RadioFieldSet extends Component {
...
+                return (
+                  <div>
+                    <div className="govuk-radios__divider">or</div>
```
Comment (Bhavesh-Patel) [inline compoments/radioFieldSet.js:61]: This block (lines 62-70) and the block in the else clause (lines 75-83) are equivalent.
Try to refactor to remove duplication of code.

File: compoments/radioFieldSet.js (hunk 2 above)
Comment (Bhavesh-Patel) [inline compoments/radioFieldSet.js:58]: This check is performed for every item in `options`, but will only be true once - for the last item.
Is it possible to improve the flow to remove the redundant checks?

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
The tests are brittle due to reliance upon copy/displayed text for finding elements.
Can they be made less brittle? Perhaps using test id's?

Think we've discussed previously, but wanted to revisit in case anything had changed.

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**